pyfiglet/__init__.py

```diff
@@ -57,11 +57,23 @@
 
 RESET_COLORS = b'\033[0m'
 
-if sys.platform == 'win32':
-    SHARED_DIRECTORY = os.path.join(os.environ["APPDATA"], "pyfiglet")
-else:
-    SHARED_DIRECTORY = '/usr/local/share/pyfiglet/'
-
+SHARED_DIRECTORIES = [
+    '/usr/share/figlet',
+    '/usr/share/pyfiglet',
+    '/usr/local/share/figlet',
+    '/usr/local/share/pyfiglet',
```

Comment:
Probably makes more sense to have the `local` versions have a higher priorities than the distro paths.

Reply:
thanks! and done